.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+# For details on how this file works refer to:
+#  - https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  #  - Check for updates once a week
+  #  - Group all updates into a single PR
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      all-actions:
+        patterns: [ "*" ]
```

Comment:
@rajpalc7, There are no GitHub actions defined for this repo.

Reply:
Thanks @WadeBarnes changes made accordingly